packages/conduit/conduit.3.0.0/opam

```diff
@@ -39,3 +39,4 @@ url {
     "sha512=bf8e996276ca9d9393e90f718392916e3b29a56817c38d927ee87a9e81ce608b22dc3e4544fc05077e516f91511b2a96e560d9e7ab917034ba0c23cf61b10f66"
   ]
 }
+available: false
```

Comment:
We need to set also all the other `conduit-*.3.0.0` as available false, otherwise they give very cryptic error message for packages with bad bounds

Reply:
Ok, I hoped that it was not the case, I will do that after the result of the CI 👍